hypertrace-view-generator/helm-test/values.yaml

```diff
@@ -0,0 +1,154 @@
+replicaCount: 1
+maxUnavailable: 0
+
+images:
+  pullPolicy: IfNotPresent
+  creator:
+    repository: hypertrace/hypertrace-view-creator
+    tagOverride: "0.4.2"
+  generator:
+    repository: hypertrace/hypertrace-view-generator
+    tagOverride: "0.4.2"
+
+imagePullSecrets: []
+
+containerAdminPort: 8099
+
+nodeLabels: {}
+
+viewCreatorJobConfig:
+  name: view-creator-job-config
+  data:
+    application.conf: |-
+      pinot.retentionTimeValue = 5
+      pinot.retentionTimeUnit = DAYS
+
+allViews:
+  creator:
+    jobName: all-views-creation-job
+    javaOpts: "-XX:InitialRAMPercentage=50.0 -XX:MaxRAMPercentage=75.0"
+    resources:
+      limits:
+        cpu: 1.0
+        memory: 384Mi
+      requests:
+        cpu: 0.1
+        memory: 384Mi
+
+logConfig:
+  name: view-generator-log-config
+  monitorInterval: 30
+  rootLogger:
+    level: INFO
+  appender:
+    rolling:
+      enabled: false
+
+groups:
+  - name: multi-view-generator-group1
```

Comment:
Can we have a name as `all-views-generator-group`?
And we just provide commented/readme for creating a separate group - 1. for service-call-view, raw-service-view and 2. backend view, span-event-view, raw-trace-view